Lib/fontmake/font_project.py

```diff
@@ -54,6 +55,7 @@
     GLYPHS_PREFIX + "customParameter.InstanceDescriptorAsGSInstance.Keep Glyphs"
 )
 GLYPH_EXPORT_KEY = GLYPHS_PREFIX + "Glyphs.Export"
+COMPAT_CHECK_KEY = GLYPHS_PREFIX + "customParameter.GSFont.Enforce Compatibility Check"
```

Comment:
How about we also add an --check-compatibilty CLI flag (False by default, ignored for interpolatable outputs [always True by definition for those])? That'd make it easier to use from non-Glyphs.app workflows, as that lib key is pretty long and obscure.

Reply:
Actually this solves the above problem too. If we create a `check_compatibility` option then we can fill it from the Glyphs file instead of grubbing around in lib files.

Reply:
> we can fill it from the Glyphs file

sure but how do you fill a CLI option from the Glyphs file exactly?